vosekast_control/I2cInputAndRelayControl.py

```diff
@@ -0,0 +1,70 @@
+import smbus
+from typing import Dict, List
+
+
+class RelayControl:
+    def __init__(self):
+        self.address_relays = 0x38
+        self.bus = smbus.SMBus(1)
+        self.state_dict = {}
+        self.state_reading = None
```

Comment:
what is the difference between `state_reading` and `binary`?

Reply:
`state_reading` is the state as read from i2c. `binary` is the current state of the relays as stored in the property. Might differ when restarted. And is hence not the same.